sema/errors.go

```diff
@@ -3358,7 +3358,7 @@ func (*InvalidMoveOperationError) Error() string {
 
 func (*InvalidMoveOperationError) SecondaryError() string {
 	return "the move operator (`<-`) can only be used on resources; " +
-		"remove the `<-` operator for the non-resource"
+		"use the assignment operator (`=`) for the non-resource"
```

Comment:
This error is reported for the unary move, e.g. in `f(<-s)`, where `s` is a struct and not a resource. The operator should be removed.

```suggestion
		"remove the `<-` operator for the non-resource"
```